packages/protobuf/messages.json

```diff
@@ -4989,6 +5031,13 @@
                 "firmware_header": {
                     "type": "bytes",
                     "id": 2
+                },
+                "language_data_length": {
+                    "type": "uint32",
+                    "id": 3,
+                    "options": {
+                        "default": 0
+                    }
```

Comment:
what is this field @Hannsek btw?

Reply:
🤷🏻‍♂️ @grdddj @matejcik

Reply:
length of language blob update. we'll tell you on Monday